app/views/episodes/_form_status.html.erb

```diff
@@ -24,32 +24,34 @@
     </div>
   </div>
 
-  <div class="card-footer">
+  <div class="card-footer d-flex align-items-center justify-content-between">
```

Comment:
This ensures the buttons all remain in alignment and properly spaced with the status text.